custom-routes-vis/.settings/org.eclipse.jdt.apt.core.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
```

Comment:
I don't think we need those files. Are you using eclipse? Maybe add them to gitignore